lib/cartopy/_crs.pyx

```diff
@@ -336,14 +336,14 @@ cdef class CRS:
             result[:, 2] = z
 
         if src_crs.is_geodetic():
-            result = np.deg2rad(result)
+            result[:, :2] = np.deg2rad(result[:, :2])
```

Comment:
I know this was the way it was written originally, but I feel like it makes more sense to do this when `result` is assigned a couple lines up. That should limit some extraneous copying.